Jenkinsfile

```diff
@@ -18,21 +18,17 @@ pipeline {
                 }
             }
         }
-        stage('Clone repository') {
-            steps {
-              sh "mkdir -p ${WORKING_DIR}"
-              sh "git clone https://github.com/kiegroup/kogito-cloud-operator.git ${WORKING_DIR}"
-            }    
-        }
         stage('Initialize') {
             steps {
+                sh "mkdir -p ${WORKING_DIR} && cd ${WORKSPACE} && cp -Rap * ${WORKING_DIR}"
```

Comment:
What is stored in WORKSPACE?

Reply:
It is the source code of the kogito operator. By default, jenkins clones this to "WORKSPACE" but  for the build to work it needs to be in that specific path. So I'm copying the code to the go path.

Reply:
Ok, thanks